packages/tools/pluggable-widgets-tools/configs/tsconfig.base.json

```diff
@@ -1,39 +1,28 @@
 {
-  "compilerOptions": {
-    "noEmitOnError": true,
-    "sourceMap": true,
-    "module": "amd",
-    "target": "es5",
-    "lib": [
-      "es2015",
-      "dom"
-    ],
-    "types": [
-      "big.js",
-      "jest"
-    ],
-    "moduleResolution": "node",
-    "declaration": false,
-    "removeComments": true,
-    "noLib": false,
-    "watch": false,
-    "forceConsistentCasingInFileNames": true,
-    "noFallthroughCasesInSwitch": true,
-    "strict": true,
-    "strictFunctionTypes": false,
-    "skipLibCheck": true,
-    "noUnusedLocals": true,
-    "noUnusedParameters": true,
-    "jsx": "react",
-    "jsxFactory": "createElement",
-    "allowSyntheticDefaultImports": true,
-    "esModuleInterop": true
-  },
-  "include": [
-    "../../../../src"
-  ],
-  "exclude": [
-    "../../../../dist/",
-    "../../../../node_modules/"
-  ]
+    "compilerOptions": {
+        "noEmitOnError": true,
+        "sourceMap": true,
+        "module": "es2015",
```

Comment:
I see that many tsconfigs override `module` to `es2015` - something which is not needed anymore.

Reply:
Good point! I removed the overrides.